proto/context/src/main/proto/spine/validation/views.proto

```diff
@@ -160,3 +161,19 @@ message ChoiceOneof {
     // The error message template.
     string error_message = 3;
 }
+
+// A view of a message that is marked with `(require)` option.
+message RequireMessage {
+    option (entity).kind = PROJECTION;
+
+    protodata.TypeName id = 1;
+
+    // The error message template.
+    string error_message = 2;
+
+    // A list of field groups as specified by a user.
+    string groups_definition = 3;
+
+    // A list of field groups.
```

Comment:
Please expand this description so that the difference between this field and the one above is clearer.